R/tribble.R

```diff
@@ -27,19 +27,75 @@
 #'   "b", 4:6
 #' )
 tribble <- function(...) {
+  data <- extract_frame_data_from_dots(...)
 
+  if (length(data$frame_rest) == 0) {
```

Comment:
Later in `extract_frame_data_from_dots()`, `frame_rest` is elegantly returned as a list with `frame_rest <- dots[-seq_along(frame_names)]`. In the empty case, this returns `frame_rest` as `list()`. I think it's better to keep `frame_list` consistently as a list no matter empty or not. 

If we remove the special case here, we have to add a special case in `extract_frame_data_from_dots()` so that `frame_rest` is `logical()`.